.omeroci/app-deps

```diff
@@ -3,5 +3,5 @@
 set -e
 set -u
 yum -y install epel-release
-curl --silent --location https://rpm.nodesource.com/setup_12.x | sudo bash -
+curl --silent --location https://rpm.nodesource.com/setup_14.x | sudo bash -
```

Comment:
you tried with 18 and it fails
16 is the closest LTS so  I think we should try with 16